If VERSION is taken from 'git describe', the filename in the tarball
can be longer than 99, which causes an error with tar v7.

diff --git a/gettext-runtime/intl-csharp/Makefile.am b/gettext-runtime/intl-csharp/Makefile.am
index c39e118..cbe2214 100644
--- a/gettext-runtime/intl-csharp/Makefile.am
+++ b/gettext-runtime/intl-csharp/Makefile.am
@@ -76,9 +76,7 @@ DOC_FILES = \
   csharpdoc/index.html \
   csharpdoc/namespaces.html \
   csharpdoc/begin.html \
-  csharpdoc/GNU_Gettext.html \
-  csharpdoc/GNU_Gettext_GettextResourceManager.html \
-  csharpdoc/GNU_Gettext_GettextResourceSet.html
+  csharpdoc/GNU_Gettext.html
 
 EXTRA_DIST += $(DOC_FILES)
 
